### PR TITLE
Environment update 2022.03.16

### PR DESCRIPTION
**Pull request by scottyhq**
* sliderule 1.4 (cc @tsutterley @SmithB)
* sat-search (cc @tsnow03)

### CODE REVIEW

**Comment by JessicaS11** [comment]
We need to add earthdata (`pip install earthdata`) as well. I have this on a separate branch locally - should I update this PR or add as a new one after this is merged?

**Comment by scottyhq** [comment]
> We need to add earthdata (pip install earthdata) as well. I have this on a separate branch locally - should I update this PR or add as a new one after this is merged? 

I thought we were waiting on a new release? But we can add whatever is currently on pypi here

**Comment by scottyhq** [comment]
@JessicaS11 @mikala-nsidc : trying to install earthdata, but waiting on https://github.com/nsidc/earthdata/issues/65

File: conda/conda-linux-64.lock.yml [hunk 1]
```diff
@@ -294,17 +297,17 @@ dependencies:
...
   - markdown-it-py=1.1.0=pyhd8ed1ab_0
-  - markupsafe=2.1.0=py39hb9d737c_1
+  - markupsafe=2.1.1=py39hb9d737c_0
   - matplotlib-inline=0.1.3=pyhd8ed1ab_0
   - mistune=0.8.4=py39h3811e60_1005
   - msgpack-python=1.0.3=py39h1a9c180_0
   - multidict=6.0.2=py39h3811e60_0
   - multipledispatch=0.6.0=py_0
-  - numpy=1.22.3=py39h18676bf_0
+  - numpy=1.21.5=py39haac66dc_0
```
Comment (scottyhq) [inline conda/conda-linux-64.lock.yml:310]: pinning to < 1.22

**Pull request merged**